From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ziyang Huang <hzyitc@outlook.com>
Date: Sun, 8 Sep 2024 16:40:12 +0800
Subject: [PATCH 010/119] clk: gcc-ipq5018: remove the unsupported clk
 combination for gmac

Comment out the unsupported clock combination in the frequency table
for GMAC1.

Signed-off-by: Ziyang Huang <hzyitc@outlook.com>
Signed-off-by: George Moussalem <george.moussalem@outlook.com>
---
 drivers/clk/qcom/gcc-ipq5018.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/clk/qcom/gcc-ipq5018.c b/drivers/clk/qcom/gcc-ipq5018.c
index 46da65295..111aec444 100644
--- a/drivers/clk/qcom/gcc-ipq5018.c
+++ b/drivers/clk/qcom/gcc-ipq5018.c
@@ -686,7 +686,7 @@ static const struct freq_tbl ftbl_gmac1_rx_clk_src[] = {
 	F(2500000, P_UNIPHY_RX, 12.5, 0, 0),
 	F(24000000, P_XO, 1, 0, 0),
 	F(25000000, P_UNIPHY_RX, 2.5, 0, 0),
-	F(125000000, P_UNIPHY_RX, 2.5, 0, 0),
+	/* F(125000000, P_UNIPHY_RX, 2.5, 0, 0), */
 	F(125000000, P_UNIPHY_RX, 1, 0, 0),
 	F(312500000, P_UNIPHY_RX, 1, 0, 0),
 	{ }
@@ -726,7 +726,7 @@ static const struct freq_tbl ftbl_gmac1_tx_clk_src[] = {
 	F(2500000, P_UNIPHY_TX, 12.5, 0, 0),
 	F(24000000, P_XO, 1, 0, 0),
 	F(25000000, P_UNIPHY_TX, 2.5, 0, 0),
-	F(125000000, P_UNIPHY_TX, 2.5, 0, 0),
+	/* F(125000000, P_UNIPHY_TX, 2.5, 0, 0), */
 	F(125000000, P_UNIPHY_TX, 1, 0, 0),
 	F(312500000, P_UNIPHY_TX, 1, 0, 0),
 	{ }
